modules/Quiz/resources/quiz.js

```diff
@@ -559,12 +559,19 @@
                 .click(function () {
                     $(".close-button")
                     .click(function () {
-                        interfaceElement.find(".open-question-textarea").val(holdAnswer);
-                        interfaceElement.find(".open-question-chars .chars").text(holdAnswer.length);
-                        if (holdAnswer.length !== 0) {
+                        if (!cPo.isAnswerd) {
```

Comment:
@shalommeoded  this is wrong ! 
Try the next scenario: 
Make an open-question with hint quiz 
answer the quiz with answer "11111111" and save your answer 
submit the quiz and replay 
see question again (after submit) and click "hint"  
Wrongly see edit-mode

Reply:
Fixed